service-discovery/README.adoc

```diff
@@ -31,304 +31,398 @@ WARNING: This content is generated by running npm --prefix .build run generate:a
 
 .Procedure
-. Execute
+
+. If you're not already logged in to your OpenShift cluster, log in as a user that has privileges to deploy applications in the OpenShift project created earlier in this tutorial.
++
+[source,subs="+quotes"]
+----
+$ oc login -u __<user>__ -p __<password>__ --server=__<host:port>__
+----
++
+In the preceding example, replace the values in angle brackets (`< >`) with your own values.
+
+. Ensure that the current OpenShift project is the one created earlier in this tutorial. For example:
 +
 [source]
 ----
-rhoas cluster connect --ignore-context
+$ oc project rhoas-quarkus
 ----
+
+. To deploy the Quarkus application, apply an example application template provided by Red Hat.
 +
-. You will be asked to select the Kafka instance you want to connect.
-Select instance and Press **Enter** to continue.
-. The CLI will print the **Connection Details** and asks you to confirm.
-. Verify if namespace that will be used to create service instance is `rhoas-quarkus`
-. Type `y` and press **Enter** to continue.
-. You will be asked to provide a token, which again can be retrieved from https://cloud.redhat.com/openshift/token . Navigate to this URL, copy the token to your clipboard, and copy it into your terminal. Press **Enter** to continue.
-. You should see the message: **KafkaConnection successfully installed on your cluster.**
-. To verify that the connection has been successfully created, execute the following **oc** command: `oc get KafkaConnection`. This should return a **KafkaConnection** with the name of your Kafka instance.
+[source,options="nowrap"]
+----
+$ oc apply -f https://raw.githubusercontent.com/redhat-developer/app-services-guides/main/code-examples/quarkus-kafka-quickstart/.kubernetes/kubernetes.yml
 
-.Verification
-ifdef::qs[]
-* `oc get KafkaConnection` returned created kafka instance
+service/rhoas-quarkus-kafka created
+deployment.apps/rhoas-quarkus-kafka created
+route.route.openshift.io/rhoas-quarkus-kafka created
+----
++
+As shown in the output, when you deploy the application, OpenShift creates a Service and Route for access to the application.
 
-In case of issues you can remove your KafkaConnection by executing
-`oc delete akc name-of-your-kafka` and retry operation
-endif::[]
+. Get the URL of the Route created for the application.
++
+[source,options="nowrap"]
+----
+$ oc get route
+
+NAME                   HOST/PORT
+rhoas-quarkus-kafka    rhoas-quarkus-kafka-jbyrne-dev.apps.sandbox-m2.ll9k.p1.openshiftapps.com
+----
+
+. In your terminal, highlight the URL shown under *HOST/PORT*. Right-click in the terminal window and  select *Copy*.
+
+. In your web browser, paste the URL for the Route.
++
+A web page for the Quarkus application opens.
+
+. In your web browser, append `/prices.html` to the URL.
++
+A new web page, entitled *Last price*, opens.  Because you have not yet connected the Quarkus application to your Kafka instance, the price value appears as `N/A`.
```

Comment:
I think can be smoothened as "A new web page entitled *Last price* opens." Just a thought.